frontend/src/components/Admin/AddUser/AddUser.tsx

```diff
@@ -125,6 +128,75 @@ export default function AddUser(): JSX.Element {
     });
   }
 
+  async function uploadUsersCsv(csvFile: File | undefined): Promise<void> {
+    if (csvFile) {
+      const csv = await csvFile.text();
+      const json = await csvtojson().fromString(csv);
+      const allNetIds = allMembers.map((m) => m.netid);
+      const members: IdolMember[] = [];
+      if (json[0].email) {
+        json.forEach((m) => {
+          const currMember = allMembers.find((mem) => mem.netid === getNetIDFromEmail(m.email));
+          if (currMember) {
+            members.push({
+              netid: getNetIDFromEmail(m.email),
```

Comment:
Since you already calculated `getNetIDFromEmail(m.email)` in `currMember`, and you do it for every member, and you use it again here, how about you store that value in a variable first so we don't have to repeatedly calculate it?